src/components/index.tests.tsx

```diff
@@ -13,6 +14,35 @@ import { ZipInput } from './index'
 
 // -------------- Localized tests ----------------//
 
+class ButtonSelectGroupTest extends React.Component<any, any> {
```

Comment:
I just moved ButtonSelectGroupTest and CheckboxInputTest up to be in alphabetical order